package.json

```diff
@@ -194,6 +194,10 @@
         "title": "Close current environment",
         "category": "LaTeX Workshop"
       },
+      {
+        "command": "latex-workshop.toggle-equation-envname",
+        "title": "Toggle between \\[...\\] and \\begin{}...\\end{}"
+      },
       {
```

Comment:
Add `"category": "LaTeX Workshop"`. 
Every function provided by the extension should be put in this category.